include/swift/IDE/CodeCompletionStringBuilder.h

```diff
@@ -0,0 +1,416 @@
+//===--- CodeCompletionStringBuilder.h ----------------------------------===//
```

Comment:
Does this actually need to be in `include/swift/IDE` vs an implementation detail in `lib/IDE` similar to the result builder?